src/frontend/screens/ObservationFields/index.tsx

```diff
@@ -89,17 +101,22 @@ const DetailsHeaderRight = ({questionNumber}: {questionNumber: number}) => {
   const navigation = useNavigationFromRoot();
   const {usePreset} = useDraftObservation();
   const preset = usePreset();
+  const observationId = usePersistedDraftObservation(
```

Comment:
Is there a reason you can't pass the observationId as a prop from above?
And the preset?

Reply:
No reason. I just used the hook for convenience!